data/sql/updates/pending_db_world/rev_XXXXXXXXXXXX.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('XXXXXXXXXXXX');
```

Comment:
I don't understand what/if I should edit the XXXXXXXXXXX part ^^ 

(Same with the name of the file)

Reply:
@Kitzunu you are not supposed neither edit that line nor to have XXXXX as a value. I guess you skipped a step:

![image](https://user-images.githubusercontent.com/75517/76108103-3da5b280-5fda-11ea-89ca-b6328a786f51.png)

I think you forgot to execute the file `./data/sql/updates/pending_db_world/create_sql.sh` which will generate the sql file for you, containing the right value

Reply:
Ah yes

Reply:
this is really easy, just run the create_sql.sh with bash (bash here or double click it) and copy the number and replace XXXXXXXXX by that number